lib/graphql/helpers/authorization.ex

```diff
@@ -148,4 +149,14 @@ defmodule Accent.GraphQL.Helpers.Authorization do
       authorize(action, integration.project_id, info, do: func.(integration, args, info))
     end
   end
+
+  def comment_authorize(action, func) do
+    fn _, args, info ->
+      comment =
+        Comment
+        |> Repo.get(args.id)
```

Comment:
We don’t have the "SinglePipe" credo check in the project but we try to not use a `|>` operator on a single function call.

```suggestion
      comment = Repo.get(Comment, args.id)
```